### ISSUE TITLE
Angular Ivy: Kendo UI packages fail to compile

### ISSUE THREAD

**Issue by tsvetomir** [stackblitz/core#1229]
Getting the following error when attempting to add the Kendo UI Grid to an Ivy project on StackBlitz, [see sample](https://stackblitz.com/edit/angular-ivy-aqigbg?file=src%2Fapp%2Fapp.module.ts):
> ngcc failed to run on @progress/kendo-angular-dateinputs@4.2.2, @progress/kendo-angular-grid@4.7.1.

The packages work fine with Ivy in a regular CLI project. Any idea what might be wrong?

**Comment by EricSimons** [comment]
@tsvetomir thanks for letting us know about this- having folks on our side investigate now 👍

**Comment by rezonant** [comment]
This is happening for one of my own Angular libraries -- is there any way we can get more info than just "ngcc failed to run"? The same library seems to work fine in a local Angular CLI app...

**Comment by EricSimons** [comment]
Yeah, the issue is that ngcc fails out on our serverless endpoint, but this can happen for a variety of reasons (package too big, peerDependencies missing, etc). @rezonant can you link your package here as well so I can test it out?

**Comment by rezonant** [comment]
Absolutely, my Angular component library is `diazo` on NPM, and here is a Stackblitz with Diazo, ngx-monaco-editor and the other peer deps loaded in that causes the issue:
https://stackblitz.com/edit/angular-ivy-vvwp1d

**Comment by eestein** [comment]
I've got the same problem with my lib. It's working nicely and being verified by `ngcc` locally. Two production apps on angular v9 are using it without any issues, but it breaks on StackBlitz.

I even created an issue on `ngcc` validation repo https://github.com/angular/ngcc-validation/issues/1384 and they confirmed my lib is properly set. Any news regarding this issue?

Thanks.

**Comment by ocombe** [comment]
Same issue with my library Locl: https://l.staticblitz.com/ngcc/v3/10.0.5/@locl/core@1.0.0 :-/
Any chance to get the complete log @EricSimons please?

**Comment by rezonant** [comment]
@ocombe Can you give some more detail? My lib still fails, if I can get that one working maybe I can PR for ngx-monaco-editor as well

**Comment by ocombe** [comment]
I had a peer dependency for ^9.0.0 and I was trying to use it in stackblitz with v10.0.5, I changed the peer dependency of the lib to ^10.0.0 and it works in stackblitz with v10.0.5 now

**Comment by rezonant** [comment]
I see, will have to explore that. While I was waiting for a response I snooped your lib and your commit to fix in case it's helpful for anyone else, it's https://github.com/loclapp/locl/commit/71001b1321841738cecfeafee304f1fb4e686e87

**Comment by tsvetomir** [comment]
Currently `ngcc` seems to pass successfully for some of the smaller packages. The [test application](https://stackblitz.com/edit/angular-ivy-eavbjt?file=src%2Fapp%2Fapp.module.ts) fails to run with a different error:

```
Import error, can't find files:
./main
```

**Comment by vthinkxie** [comment]
meet the same problem when enabling ivy in ng-zorro-antd https://stackblitz.com/edit/ng-zorro-antd-ivy?file=src%2Fmain.ts
